package.json

```diff
@@ -4,7 +4,7 @@
     "lint": "eslint --ext .js,.mjs,.ts,.vue .",
     "linkinator": "linkinator dist/ --recurse --verbosity error --skip 'twitter.com|www.farmlands.co.nz|www.instagram.com|yourapp.example.com|centrapay-docs.dev.s3-website-ap-southeast-1.amazonaws.com/api'",
     "dev": "astro dev",
-    "build": "astro build",
+    "build": "./scripts/create-code-snippets.js && astro build",
```

Comment:
@MeganSteenkamp Suggestion: Integrate this directly with Astro's build using the `astro:build:start` hook

https://docs.astro.build/en/reference/integrations-reference/#astrobuildstart